tests/test___init__.py

```diff
@@ -20,7 +21,7 @@ def load(model_name=model_name, layer=layer, region=region, pca_components=pca_c
             activations_model = base_model_pool[model_name]
             if pca_components:
                 LayerPCA.hook(activations_model, n_components=pca_components)
-                activations_model.identifier += Hooks.HOOK_SEPARATOR + "pca_1000"
+                activations_model.identifier += "pca_1000"
```

Comment:
let's have some separation between identifier and the pca suffix, e.g. `+= "--pca_1000"

Reply:
For consistency I made it `-pca_1000`